### ISSUE TITLE
Highlight colors

### ISSUE THREAD

**Issue by memco** [silofy/hackthebox#2]
Hi, loving HTB, but have a few issues I've seen:

Highlighting text in boxes with a dark background (search, rename files in tree) are invisible. 

`selection.background: "#A4B1CD"` 

Seems to be a reasonable fix for this.

Also, the find colors and multi-cursor don't seem to play together very well. It would be great to see some better styles applied there as well.

**Comment by barnabasmolnar** [comment]
Can confirm memco's observations. Would be nice to have these minor issues fixed. Because I'm finding it a fantastic theme otherwise, really enjoying it so far. 👍

**Comment by 3olkin** [comment]
Agree with memco's observations. Pls find the best solution for selected items colors.
Hope you ll support this awesome project

**Comment by memco** [comment]
For what it's worth I made these customizations:

```json
"workbench.colorCustomizations": {
    "editorCursor.foreground": "#ffcc5c",
    "selection.background": "#a4b1cd66",
    "editor.selection.background": "#a4b1cd66",
    "editor.findMatchBackground": "#b19f7a99",
    "editor.findMatchHighlightBackground": "#ffcc5c33",
    "editor.selectionHighlightBackground": "#6e7b9633",
    "activityBar.foreground": "#cf8dfb",
    "activityBar.inactiveForeground": "#c5d1eb88"
  },
  "editor.tokenColorCustomizations": {
    "textMateRules": [
      {
        "scope": [
          "meta.function-call.arguments.python"
        ],
        "settings": {
          "foreground": "#5cb2ff"
        }
      }
    ]
  },
```

I'm not really happy with those styles for the find & selections: they aren't pretty, but they are functional. Also threw in a brighter activity bar color and changed the active activity bar highlight so it's easier for me to see. (I have low vision and the default styles were too low contrast for the activity bar). The Python arguments is obviously unrelated, but felt like it would be worth pointing out as another change that might improve HTB by being more consistent since it looks like other languages are using blue for function args.

**Comment by silofy** [comment]
Hi all, thanks for your feedback, we really appreciate it and love to have the community involved on this project.

I will take a look at fixing:

- Highlighting text in boxes
- Find colors
- Multi cursor

@memco about the contrast issues. We will be putting together a higher contrast theme with appropriate HTB colors when we can find the time. I will take a look at what you have suggested and maybe we can improve on that.

Thanks again and keep up the effort with sending us issues 💚

**Comment by brilang** [comment]
Any updates on this?
I've just implemented the suggestions from @memco and I agree they're not as pretty as they could be.

The way I'm reading the editor.tokenColorCustomizations means this is only changing it for python function calls, right?

**Comment by silofy** [comment]
Fixed in 0.2.9, thanks for your help!
